### PR TITLE
make the notificationform a view and add field to allow redirects

### PR DESCRIPTION
**Pull request by Tschanzt**
@elioschmutz I moved the wrapper from the notification form from a cpt to the view and renamed it from notification_form_content to notification_form. Also I added a Field so we still know from where it was callled and can redirect there after sending the notification.

### CODE REVIEW

**Comment by jone** [comment]
@Tschanzt can you fix the tests?
Seems that you have removed the last file from the skins directory. 
In git terms this means there is no longer a directory. 
This leads to a ZCML error.
I think you need to remove the skins directory altogether (with an upgrade step reverting the `skins.xml`).

**Comment by Tschanzt** [comment]
@jone I updated the pr.

**Comment by jone** [comment]
Can you add some tests, please? :cat:

**Comment by Tschanzt** [comment]
@jone I added some tests.

File: ftw/notification/base/testing.py [hunk 1]
```diff
@@ -7,6 +9,12 @@
 from zope.configuration import xmlconfig
 
 
+def functional_session_factory():
```
Comment (jone) [inline ftw/notification/base/testing.py]: :scissors: 
```python
from ftw.builder.testing import functional_session_factory
```

**Comment by Tschanzt** [comment]
@jone updated.

**Pull request merged**